app/controllers/admin/organizations_controller.rb

```diff
@@ -6,10 +6,14 @@ def edit
 
   def update
     @organization = Organization.find(params[:id])
-    if OrganizationUpdateService.update(@organization, organization_params)
+    update = OrganizationUpdateService.update(@organization, organization_params)
+
+    return false unless update
+
+    if update.errors.none?
```

Comment:
This seems like a bad flow. The service should return true if the update was successful (in which case there would be no errors) and false if it was not successful (in which case there would be errors). Either way we shouldn't have an early return here. In fact, early returning will just mean "render the default view", and there is no `update.html.erb` view at all.

Reply:
Yeah, there's a weird test suite where an update does not return an organization. So that's why I put in the early return.

As for returning true and false vs the organization, I can make that happen.

And that is good to note about returning early inside controllers!